test/app/database/models/numeric-id.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const schema = mongoose.Schema({
+  _id: {
+    type: Number,
+    required: true
```

Comment:
Wouldn't `serializedId = String(unserializedId)` cover 99% of use cases? If you're thinking of providing a full serialization API, that could be useful, but surely that's a tiny, tiny edge case.